### PR TITLE
EvaSjöholm/Weather-App

### PR DESCRIPTION
**Pull request by EvaSjoholm**
A project ment for pairs and because of last minute changes, I decided to continue doing solo. 
Weather app that contains HTML, CSS, JavaScript and API (fetch-method).

### CODE REVIEW

**Comment by EvaSjoholm** [comment]
This project was tough to create regarding the JavaScript level as well as working with array's and loops in a deeper level. 
I wouldn't have managed it without help from classmates to understand the steps. 

I started in pair with a plan and decided to start with the foundation for HTML and some CSS. 
Due to last minute changes for my partner I decided to continue the project solo. I redid the whole outlook and wanted to have a dynamic and creative background with a simple yet classy look on the weather-card itself. I god inspiration from former students on how to add the text and even which weather information that was necessary from Technigo. 

I managed thanks to help from classmates, Stackoverflow and former students to reach all 3 steps. 
Last minute I decided to even get an ICON working which I am proud of. 

I love the simple yet dynamic design I created and that the weather information is working up to date.

**Comment by EvaSjoholm** [comment]
Weather App through netlify: https://glittery-smakager-2bd241.netlify.app/

File: script.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+// DOM / API
```
Comment (NinaWald) [inline script.js:1]: Very easy code to read, great!

File: README.md [hunk 2]
```diff
@@ -1,13 +1,24 @@
...
+✓ How to use promises in JavaScript
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+✓ How to work with the `Date()` object in JavaScript
 
-## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+* Problems:
+From being in pairs to ending up solo due to my partner needing to change directions privately, I decided to do the project solo.
```
Comment (NinaWald) [inline README.md:14]: Well done Eva, you did it alone!

File: style.css [hunk 3]
```diff
@@ -0,0 +1,99 @@
+body {
```
Comment (NinaWald) [inline style.css:1]: Very nice pretty design! Good css work!

File: style.css [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+  }
+  
+  #forecastFiveDayAndTemp {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-right: 50px;
+  }
+}
+  }
```
Comment (noraterbocs) [inline style.css:99]: You chose a really nice CSS design! 
I have however noticed that on bigger screen the background image breaks, you can fix it quickly by giving the html 100% width and then with flexbox you can position the footer at the bottom and everything else in the middle. 

```
html {
width:100%
}
```

**Review by noraterbocs — COMMENTED** [review]
Eva I think you can be proud of yourself because you managed to complete the task and your code is really easy to understand and super organized. You have a good sense for design, as well! Good job! 👍

**Pull request closed without merging**